install.sh

```diff
@@ -29,25 +29,30 @@ main()
     # Never empty, dirname returns "." instead (opengroup.org)
     local path; path=$(dirname "$1")
     local ver; ver=$(basename "$1")
-    local sdir sloc optversuffix
+    local sdir optversuffix
 
     [ -z "$ver" ] || optversuffix="-$ver"
 
-    for sdir in sof sof-tplg; do
-        sloc="$path/${sdir}${optversuffix}"
-        test -d "$sloc" ||
-            die "%s not found\n" "$sloc"
-    done
-
     # Do this first so we can fail immediately and not leave a
     # half-install behind
     if [ -n "$optversuffix" ]; then
     set -x
-    for sdir in sof sof-tplg; do
-        ln -sT "$sdir-$ver" "${FW_DEST}/$sdir" || {
-            set +x
-            die '%s already installed? (Re)move it first.\n' "${FW_DEST}/$sdir"
-        }
+
```

Comment:
Please fix the comment above to mention symbolic links. That miss was not great  when the code that small but now there's a lot more code to read. Plus the `ln` line has never been very readable because of all the variables.

Reply:
@marc-hb I was wondering why you kept the intendation. Now fixed.

Reply:
I meant the source code comment when you're reading the code but not running it

Reply:
Aa, right. I think that is close enough. I agree we shouldn't expand this anymore, so if anything. I'd just remove the install exists checks (and make install.sh closer to the one-liner-run-rsync that is supposed to be in the end).